newm-server/src/main/kotlin/io/newm/server/typealiases/SongId.kt

```diff
@@ -0,0 +1,5 @@
+package io.newm.server.typealiases
+
+import java.util.*
```

Comment:
do we need this import?

Reply:
It should be `import java.util.UUID` , need to fix import setup in IJ